### ISSUE TITLE
Typo in stackPrefetch.js?

### ISSUE THREAD

**Issue by freund17** [chafey/cornerstoneTools#211]
Line 135:
```JavaScript
[...]
while (lowerIndex > 0 || higherIndex < stackPrefetch.indicesToRequest.length) {
    if (lowerIndex >= 0) {
[...]
```

Looks like it should be:
```JavaScript
[...]
while (lowerIndex >= 0 || higherIndex < stackPrefetch.indicesToRequest.length) {
    if (lowerIndex >= 0) {
[...]
```

**Comment by diego0020** [comment]
I think you are right

**Comment by swederik** [comment]
This has now been fixed in 1773ac9808b09bbbae2ac8b61ba0906f104d4991

Thanks!
